### ISSUE TITLE
Block-Level Font Family changes not working on sites using Classic Themes

### ISSUE THREAD

**Issue by mxhassani** [Automattic/wp-calypso#82364]
### Quick summary

Changing the font of a specific block doesn't reflect in editor or in front end.

### Steps to reproduce

1. Activate the theme Barnsbury
2. Edit a page and select a block to edit
3. In the right sidebar, change the font of the block 

### What you expected to happen

The font to change in both the editor and in the front end.

### What actually happened

The font doesn't change in the front end.
The font changes in Editor in simple sites, but not in Atomic sites.

### Context

6754208-zen

### Platform (Simple, Atomic, or both?)

Simple, Atomic

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

Yes, difficult to implement

### Workaround details

Suggesting a switch to barnsbury23 can be a workaround.

**Comment by cuemarie** [comment]
📌 REPRODUCTION RESULTS
- Tested on Simple – **Replicated**
- Tested on Atomic – **Replicated** 
- Replicable outside of Dotcom –No 

📌 FINDINGS/SCREENSHOTS/VIDEO

Replicated on Atomic and Simple on all classic themes. Themes tested:

- Barnsbury
- Varia
- Twenty Twenty
- Twenty Twenty One

**Atomic**

Barnsbury:

https://github.com/Automattic/themes/assets/27249804/3599a65e-3426-4db3-b8cc-1befba0dca29

Varia:

https://github.com/Automattic/themes/assets/27249804/5c7f6c71-c588-42d3-a060-e77af2e7a597

Twenty Twenty:

https://github.com/Automattic/themes/assets/27249804/1b3dfcc5-91fb-41ce-9be3-9d673d8e0ec4

**Test Environments**
_Simple_
- Gutenberg: v16.6.0
- Editing Toolkit: 3.79835
- WP AMP: 2.0.5

_Atomic_
- Gutenberg: v16.6.0
- ETK: Version 3.79835

📌 ACTIONS
- Transferred to wp-calypso
- Updated details of issue
- Triaged

**Comment by cuemarie** [comment]
Escalated here: p1695932887101299-slack-C01LXBU1N21

**Comment by arthur791004** [comment]
Tested this issue again, and it seems to be resolved.

### Page Editor

https://github.com/Automattic/wp-calypso/assets/13596067/0a87c0fa-a32a-448d-8039-45836e8709ff

### Preview

![image](https://github.com/Automattic/wp-calypso/assets/13596067/9bde948b-4046-47d6-8ccc-67110455a5d8)

**Comment by arthur791004** [comment]
@mxhassani Could you help to confirm it again? Thanks!

**Comment by arthur791004** [comment]
Closing it as the issue is resolved.

**Comment by jerrysarcastic** [comment]
Is it possible that there has been a regression here?  On my simple test site I am able to set block-level custom fonts, and they are rendered correctly in the editor, but on the front-end a fallback font is shown instead.

Tested on simple WordPress.com and Atomic.  Could not replicate on self-hosted.

Steps to replicate:
- set site to use classic theme (I used Dyad 2)
- add text block to layout, can be paragraph or heading block
- set block to use a custom font, note that the font is shown correctly in the editor
- publish and visit page front-end

Expected:  Custom font to be displayed

What happened:  Custom font is not displayed and fallback font (generic Sans Serif font) is shown instead.

Reported in 9166049-zd-a8c
